_assets/ci/common.groovy

```diff
@@ -0,0 +1,27 @@
+def getFilename(path) {
+  return path.tokenize('/')[-1]
+}
+
+def uploadArtifact(path) {
+  /* defaults for upload */
+  def domain = 'ams3.digitaloceanspaces.com'
+  def bucket = 'status-im'
+  withCredentials([usernamePassword(
+    credentialsId: 'digital-ocean-access-keys',
+    usernameVariable: 'DO_ACCESS_KEY',
+    passwordVariable: 'DO_SECRET_KEY'
+  )]) {
+    sh """
+      s3cmd \\
+        --acl-public \\
+        --host='${domain}' \\
+        --host-bucket='%(bucket)s.${domain}' \\
```

Comment:
Why not `${bucket}.${domain}`?

Reply:
Good question, this came from @jakubgs, so I'll let him explain the reasoning.

Reply:
Both would work, but this way was the one I saw in examples of people using `s3cmd`, so this is the one I used.